src/Core/pages/year_change.py

```diff
@@ -91,7 +92,7 @@
             dcc.Checklist(
                 id="event-checklist",
                 options=[{'label': html.Span(i, style={'padding-left': 10}), 'value': i} for i in eventsList],
-                value=[eventsList[0]],
+                value=eventsList,
```

Comment:
Like previous comment: df["Event Type Group"].unique

Reply:
i find it nice to reuse the variable, since i do not know whether pandas caches the value of unique. In my opinion it is more readable this way.

Another side note is that i discussed with Kevin that we should probably remove the entire checklist since the functionality is built into plotly itself by clicking on the legend. 
The only downgrade of this is that the filtering is not kept between toggles of the radio button